operator/v1/types_ingress.go

```diff
@@ -332,6 +347,10 @@ type IngressControllerStatus struct {
 	//     * DNS records have been successfully created.
 	//   - False if any of those conditions are unsatisfied.
 	Conditions []OperatorCondition `json:"conditions,omitempty"`
+
+	// tlsProfile is the TLS connection configuration that is in effect.
```

Comment:
When you have multiple pods at different TLS profiles, what value will this show me?

Reply:
I'm not sure what you mean here, can you rephrase, maybe with an example in terms of ingresscontrollers?

Reply:
ingresscontroller leads to 3 pods total.  user changes cipher suites and you being rolling out.  As you replace one pod at a time, the ingresscontroller has 2 old pods and 1 new pod, then gets stuck. What is the value of this field?

Reply:
Well, it would be either the old or the new cipher config depending on how we implement the reconciliation. It sounds like you already have an idea of which it should be and why — if so can you elaborate?

If the value is the new config, and we haven't fully converged on that value, is status misleading?

If the value is the old config, and the spec value is nil (indicating the use of runtime defaults), status doesn't reflect the state of the new pods until full convergence — any problems there?

Reply:
@deads2k I need a reply to this. Are you suggesting a specific change? If not, I need to know so we can move on.

Reply:
> Well, it would be either the old or the new cipher config depending on how we implement the reconciliation.

Could be a third value too:

1. `spec` and `status` at A, all pods at A, life is good.
2. Bump `spec` to B, some pods upgrade to B.
3. Bump `spec` to C, some pods upgrade to C.  Still have some pods at both A and B.

But if we bump the `status` value when we complete a rollout, then folks can use "`status` diverges from `spec`" to see that that particular value is still progressing, and Progressing status could be exposed in a human-readable way in a Progressing condition.  So I don't think this blocks this PR unless we want to bump the docs to change "is in effect" -> "was most recently reconciled" or some such.